packages/fx-core/src/component/generator/utils.ts

```diff
@@ -209,10 +218,14 @@ function escapeEmptyVariable(
       token[0] = "text";
       token[1] = tags[0] + value + tags[1];
       accShift += shift;
+    } else if (token[0] === "#") {
```

Comment:
Could you add a unit test for this logic? 
An example: https://github.com/OfficeDev/TeamsFx/blob/2dd9386184e52c0d8531dbd4a20ea6c8087481ae/packages/fx-core/tests/component/generator/generator.test.ts#L658-L670

Reply:
Good point, added.